sources/packages/api/src/utilities/string-builder.ts

```diff
@@ -0,0 +1,67 @@
+import * as dayjs from "dayjs";
+
+/**
+ * Allow the construction of a large string piece by
+ * piece with some useful methods to help the
+ * string manipulation.
+ */
+export class StringBuilder {
+  private readonly contents: string[];
+  constructor() {
+    this.contents = [];
+  }
+
+  /**
+   * Appends a string to the current content.
+   * @param s
+   */
+  public Append(s: string) {
```

Comment:
why method names of this call all capital?

Reply:
Changed!